Makefile

```diff
@@ -8,33 +8,35 @@ build:
 clean:
 	docker-compose down -v --remove-orphans
 
+cypress:
+	# Turn on local system, and open cypress in interactive mode
```

Comment:
Should probably comment that `npx` needs to be installed on the local machine, since this doesn't use the cypress docker images.